lms/djangoapps/bulk_enroll/tests/test_views.py

```diff
@@ -65,7 +65,7 @@ def setUp(self):
                                                last_name='Student')
 
         # Email URL values
-        self.site_name = microsite.get_value(
+        self.site_name = get_value(
```

Comment:
Nit: in order to have this line of code be self-descriptive on its own, you can import `get_value` above as `get_site_value` so future developers who read this line can quickly understand that we're getting a site-specific value.